core/src/typ.rs

```diff
@@ -818,6 +821,7 @@ impl Subcontract for Type {
             TypeF::Number => internals::num(),
             TypeF::Bool => internals::bool(),
             TypeF::String => internals::string(),
+            TypeF::Opaque => internals::opaque(),
```

Comment:
I think this contract can never be instantiated in practice (unless programmatically) . For example, we don't even bother elaborating a contract for `Symbol` below (which is also arguably not a very useful type). Maybe it would be more useful to have a `std.contract.Opaque` contract?